Tasks/Shkred/Task_1/task_1.py

```diff
@@ -0,0 +1,30 @@
+def validate_input(text, type):
+    flag = False
+    while not flag:
+        inp = input(text +'\n')
+        try: 
+            if type == 'int': 
+                result = int(inp)
+            elif type == 'float': 
+                result = float(inp) 
+            flag = True           
+        except:
+            flag = False
+    return result
```

Comment:
this is a good function, but it's a money-related task, so I advise to use the Decimal data type for the calculation